Rudder-Branch/Classes/RudderUtils.m

```diff
@@ -0,0 +1,76 @@
+//
+//  RudderUtils.m
+//  Rudder-Branch
+//
+//  Created by Abhishek Pandey on 23/11/23.
+//
+
+#import <Foundation/Foundation.h>
+#import "RudderUtils.h"
+
+@implementation RudderUtils
+
++ (BOOL)isEmpty:(NSString *)value {
+    return value == nil || [value length] == 0;
+}
+
++(NSString*)truncateUserIdIfExceedsLimit:(NSString*)userId andLimit:(int)limit{
+    if([userId length] > limit) {
+        userId = [[userId substringToIndex:limit] mutableCopy];
+    }
+    return userId;
+}
+
++(BNCProductCategory) getProductCategory:(NSString*)category {
+    if (category == nil) {
+        return nil;
+    }
+    
+    if ([category isEqualToString:@"Animals & Pet Supplies"]) {
```

Comment:
Can we use switch over if-else here ?

Reply:
I tried, but Switch in objective-c doesn't work with String, refer here: https://stackoverflow.com/a/19067898.

Reply:
got it.